createmobilespec/createmobilespec.js

```diff
@@ -424,7 +430,7 @@ if (!argv.skiplink) {
         // so they actually get tested instead of downloading the last published
         // one from the npm registry. Fail if they are not.
         console.log("You are on master branch of tools, checking npm links");
-        verifyNpmLinkOf("cordova-js", "cordova-lib");
+        //verifyNpmLinkOf("cordova-js", "cordova-lib"); TODO cordova-js is not a dependency of cordova-lib (any more?)
```

Comment:
I raised PR #174 to remove this line along with some other lines related to cordova-js. No need for TODO item here.

Reply:
In the context of this PR the TODO is still required as it didn't do this cleanup, but skipped the blocking check.
```suggestion
        //verifyNpmLinkOf("cordova-js", "cordova-lib"); TODO Clean up other cordova-js stuff above
```